modules/combined/tests/fdp_geometric_coupling/fdp_geometric_coupling_test.i

```diff
@@ -211,4 +211,5 @@
   exodus = true
   print_linear_residuals = true
   print_perf_log = true
+  hide = 'dummy_name_contact_force_0 dummy_name_contact_force_1 dummy_name_contact_force_2'
```

Comment:
I will have to add some syntax to the AuxVariable/Variable action to add a
'outputs' parameter that can hide or limit the output of the variable.


On Thu, Mar 26, 2015 at 10:26 AM, Shane Stafford <notifications@github.com>
wrote:

> In
> modules/combined/tests/fdp_geometric_coupling/fdp_geometric_coupling_test.i
> <https://github.com/idaholab/moose/pull/4851#discussion_r27235376>:
>
> > @@ -211,4 +211,5 @@
> >    exodus = true
> >    print_linear_residuals = true
> >    print_perf_log = true
> > +  hide = 'dummy_name_contact_force_0 dummy_name_contact_force_1 dummy_name_contact_force_2'
>
> They are AuxVariables, but they do not use AuxKernels. Their values are
> managed from within MechanicalContactConstraint.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/4851/files#r27235376>.
>